http/obmc_shell.hpp

```diff
@@ -60,43 +60,15 @@ class Handler : public std::enable_shared_from_this<Handler>
 
             // sets the effective user ID
             // as service user
-            bool isUidSet = false;
             if (auto userName = session->getUserName(); !userName.empty())
             {
-                if (struct passwd* pw = getpwnam(userName.c_str());
-                    pw != nullptr)
-                {
-                    int uidr = setuid(pw->pw_uid);
-                    if (!uidr)
-                    {
-                        isUidSet = true;
-                    }
-                    else
-                    {
-                        BMCWEB_LOG_ERROR << "sets service user failed Error: "
-                                         << uidr;
-                    }
-                }
-                else
-                {
-                    BMCWEB_LOG_ERROR << "getpwnam return null passwd";
-                }
+                // create /bin/sh chiled process
```

Comment:
The child process was created by forkpty above.  The comment 4 lines above and this comment and not correct.  The execl() call logs in the service user.

Reply:
done